ortools/sat/README.md

```diff
@@ -11,7 +11,7 @@ CP-SAT was described during the CPAIOR 2020 masterclass. The recording is
 available [here](https://www.youtube.com/watch?v=lmy1ddn4cyw).
 
 To begin, skim
-[cp_model.proto](../cp_model.proto) to
+[cp_model.proto](../sat/cp_model.proto) to
```

Comment:
why not just (cp_model.proto) ?

Reply:
(and below)

Reply:
Actually, the links of other files in this sub-directory were like this (../sat/file_name), so that's why.